src/tools/RelayNativeUnstableBatchedUpdates.js

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Indeed it can, and we can get rid of what's in package.json...because the RN packager looks for the native extension. Though @spicyj has mentioned that he'd like to remove that eventually. I suppose if he does, we can just put back the package.json directive.